xml/System/ArraySegment`1.xml

```diff
@@ -1185,6 +1177,7 @@
  -or-  
   
  Type <paramref name="T" /> cannot be cast automatically to the type of the destination <paramref name="array" />.</exception>
+        <exception cref="T:System.InvalidOperationException">The underlying array is <see langword="null" />.</exception>
```

Comment:
Maybe it's just me, but I found this confusing because there are two arrays involved -- the underlying array and the target. I'd change this to "The array underlying this \<see langword="ArraySegment\<T>" /> is \<see langword="null." />.